arlo_server/sample_sizes.py

```diff
@@ -0,0 +1,55 @@
+from flask import jsonify, request
+from collections import defaultdict
+import math
+from werkzeug.exceptions import BadRequest
+from typing import Dict
+
+from arlo_server import app, db
+from arlo_server.models import Election, Contest
+from arlo_server.routes import require_audit_admin_for_organization
+from audits import bravo, sampler_contest
+
+
+@app.route("/election/<election_id>/sample-sizes", methods=["GET"])
+def get_sample_sizes(election_id: str):
+    election = Election.query.get_or_404(election_id)
+    require_audit_admin_for_organization(election.organization_id)
```

Comment:
suggestion for later improvement: a decorator that transforms `election_id` param into `election` param, returns 404 if not found, returns 403 is not right permission.

Reply:
#380